lightning/backend/importer.go

```diff
@@ -118,8 +118,9 @@ func isIgnorableOpenCloseEngineError(err error) bool {
 }
 
 func (importer *importer) OpenEngine(ctx context.Context, engineUUID uuid.UUID) error {
+	u, _ := engineUUID.MarshalBinary()
 	req := &kv.OpenEngineRequest{
-		Uuid: engineUUID.Bytes(),
+		Uuid: u,
```

Comment:
```suggestion
		Uuid: engineUUID[:],
```

(and the rest of this file)